plugins/nanospell/plugin.js

```diff
@@ -341,23 +348,36 @@
 				clearAllSpellCheckingSpans(editor.editable());
 			}
 
-			function checkNow() {
+			function checkNow(root) {
```

Comment:
"root" param is mostly unused for now.  the idea is that the emitted events would provide a root.  but we don't currently use it for anything.